### DISCUSSION TITLE
Contacting mentor

### DISCUSSION THREAD

**Discussion by psankhe28** [camicroscope/GSOC#12] [Q&A] [answered]
How can I contact the mentor? Are there any tasks for the interested contributors apart from the code challenge mentioned?

**Comment by birm** [comment, marked as answer]
We've made a basic page to try to help people get started. See https://github.com/camicroscope/GSOC/discussions

A good first step is to try to run and use the application. caMicroscope is a reasonably ambitious project for two part-time developers, so I'm usually pretty confident that you'll find something that behaves in a way that you don't expect, so documentation and/or PRs (against develop branches) are usually a good place after raising a detailed issue report.
